### ISSUE TITLE
Total Training Reward rises then drops again

### ISSUE THREAD

**Issue by annahambi** [werner-duvaud/muzero-general#176]
Dear MuZero specialists

Does one of you have a good tip for me what I might be doing wrong... 
1) What determines the length of this plateau region in the training?
2) Why does my total reward first rise, but then drop again?
 
![image](https://user-images.githubusercontent.com/73515327/135271578-a15082ee-bba3-4089-b50a-176cb8c3d1f7.png)

It would be great to get some educated guesses on the hyperparameters I should change : ) 
Anna

**Comment by ahainaut** [comment]
Hello,
Regarding plateau on MuZero, I would recommend tuning the parameters `lr_decay_rate` and `lr_decay_steps` which control the slope of the learning_rate curve. Tuning the `lr_decay_rate` should avoid strong changes while updating the weight of the networks.
Hope this helps.

**Comment by ahainaut** [comment]
Closing this for inactivity but feel free to reopen for any other additional informations !
